geode-core/src/integrationTest/java/org/apache/geode/internal/cache/OfflineSnapshotJUnitTest.java

```diff
@@ -51,8 +51,8 @@ public void testExport() throws Exception {
         Region<Integer, MyObject> region =
             rgen.createRegion(cache, ds.getName(), rt, "test" + rcount++);
         final Map<Integer, MyObject> expected = createExpected(st, 1000);
-
         region.putAll(expected);
+        removeAndInvalidateKeys(region, expected);
```

Comment:
It will be nice to create a separate test for testing tombstone (destroy). But I am fine with this too.